src/redist/targets/packaging/windows/clisdk/bundle.wxs

```diff
@@ -160,7 +160,7 @@
   <Fragment>
     <PayloadGroup Id="DotnetCoreBAPayloads">
       <Payload Name="thm.xml" Compressed="yes" SourceFile="bundle.thm" />
-      <Payload Name="thm.wxl" Compressed="yes" SourceFile="bundle.wxl" />
+      <Payload Name="thm.wxl" Compressed="yes" SourceFile="$(var.LocalizedContentDir)\bundle.wxl" />
```

Comment:
What you want here is to iterate over the LCIDs and pull in all the localized .wxl files as additional payload, so you'll have 1 bundle with 13 different copies of the thm.wxl. The default UI will figure out the OS locale and then look for a matching file and failing that, should fall back to the ENU/neutral locale.

Reply:
Done.